exampleSite/content/index/awesome-team/donald.md

```diff
@@ -0,0 +1,27 @@
++++
+  title = "Tiny Gopher 4"
+  weight = 30
+  joined = "2017-10-17"
+  image = "tinygopher.png"
+  position = "Gopherineer"
+
+  livesIn = "[Iceland](https://www.google.com/maps/place/Iceland/)"
+  scope = [
+    "Gopherineering for various tasks",
+    "Contributor for [Goper Team A](#)",
+    "Gopher Community Member"
+  ]
+
++++
+
+
+Really tiny Gopher
+
+Some more text to showcase the capabilities:
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+Curabitur a lorem urna.
+Quisque in neque malesuada, sollicitudin nunc porttitor, ornare est.
+Praesent ante enim, bibendum sed hendrerit et, iaculis laoreet felis.
+Morbi efficitur dui sit amet orci porttitor, nec tincidunt turpis elementum.
+
+ 
```

Comment:
Whitespace unnecessary.

Reply:
I don't know how this happens every time. Can you recommend any whitespace extension for vscode or sublime?